sql/core/src/test/resources/decimal.csv

```diff
@@ -0,0 +1,4 @@
+decimal
+2160273033000006010010000035858
```

Comment:
Would be good to add another column with large decimal and one double value. The type then should be Double.